packages/cx/src/data/comparer.js

```diff
@@ -18,12 +18,17 @@ export function getComparer(sorters, dataAccessor, comparer) {
          d = data[i];
          av = d.getter(a);
          bv = d.getter(b);
+
+         // show nulls always on the bottom
+         if (av == null) return 1;
+         if (bv == null) return -1;
```

Comment:
For stability reasons, I think there should be a check if both values are null and in that case the result should be 0.